### DISCUSSION TITLE
Brand new app - Can't perform a React state update on an unmounted component

### DISCUSSION THREAD

**Discussion by papadi** [infinitered/ignite#1941] [General]
When I'm creating a new app (npx ignite-cli new MyTestApp --expo) and run it on the browser without any modification of my own (yarn web) then I get this error in my browser console.

"Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function."

Any idea why? I cannot detect where it's coming from.

**Comment by jamonholmgren** [comment]
I've replicated this issue. Moved to issue here: https://github.com/infinitered/ignite/issues/1942

<img width="682" alt="terminal" src="https://user-images.githubusercontent.com/1479215/168321055-57769ff4-3143-4be6-b2f7-404a9db5860b.png">
<img width="1248" alt="browser" src="https://user-images.githubusercontent.com/1479215/168321063-8ff8947f-84bc-496a-8a45-c094478f5063.png">
